lib/roo_on_rails/checks/git/origin.rb

```diff
@@ -0,0 +1,24 @@
+require 'roo_on_rails/checks/base'
+
+module RooOnRails
+  module Checks
+    module Git
+      class Origin < Base
+        def _intro
+          "Checking your Git origin remote..."
+        end
+
+        def _call
+          status, url = _run "git config remote.origin.url"
+          _fail "Origin does not seem to be configured." unless status
+          
+          org, repo = url.strip.gsub(%r{\.git$}, '').split(%r{[:/]}).last(2)
```

Comment:
Could just use `sub` here as it's anchored to the end. Also I believe it's considered better to use `\Z` rather than `$` in Ruby unless you specifically mean the end of a line in a multiline string?